controlplane/kubeadm/internal/workload_cluster_coredns.go

```diff
@@ -49,7 +49,7 @@ const (
 
 	// kubernetesImageRepository is the default Kubernetes image repository for build artifacts.
 	kubernetesImageRepository    = "registry.k8s.io"
-	oldKubernetesImageRepository = "k8s.gcr.io"
+	oldKubernetesImageRepository = "registry.k8s.io"
```

Comment:
I think we want to keep this value to `k8s.gcr.io` since we `kubernetesImageRepository` to represent the new registry.

Reply:
Thanks @ameukam! I agree that was a mistake. Fixed